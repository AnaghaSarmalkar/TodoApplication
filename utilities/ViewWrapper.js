import React from "react";
import { View } from "react-native";

export default function ViewWrapper(props) {
  return (
    <View
      {...props}
      // style={{
      //   flex: 1,
      //   alignItems: "center",
      //   justifyContent: "center",
      //   // paddingTop: insets.top,
      // }}
    >
      {props.children}
    </View>
  );
}
