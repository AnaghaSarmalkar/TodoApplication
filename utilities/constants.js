import { Platform } from "react-native";

export const isIos = Platform.OS === "ios";
export const isWeb = Platform.OS === "web";
export const isAndroid = Platform.OS === "android";
