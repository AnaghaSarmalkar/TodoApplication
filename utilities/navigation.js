export function setPageTitle(navigation, title) {
  if (title) {
    navigation.setOptions({
      title: title,
    });
  }
}
