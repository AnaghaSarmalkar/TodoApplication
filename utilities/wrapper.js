import React from "react";
import { SafeAreaView } from "react-native-safe-area-context";

function SafeAreaWrapper({ children }) {
  <SafeAreaView>{children}</SafeAreaView>;
}
