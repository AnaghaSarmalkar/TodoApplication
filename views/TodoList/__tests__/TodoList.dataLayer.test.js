import React from "react";
import { shallow } from "../../../utilities/testing";
import TodoListDataLayer from "../TodoList.dataLayer";

jest.mock("@apollo/client", () => ({
  gql: jest.fn(),
  useQuery: jest.fn(() => ({})),
}));

jest.mock("react-native-safe-area-context", () => ({
  useSafeAreaInsets: jest.fn(() => ({})),
}));

jest.mock("@react-navigation/native", () => ({
  useNavigation: jest.fn(() => ({
    navigate: jest.fn(),
    setOptions: jest.fn(),
  })),
}));

describe("TodoListDataLayer", () => {
  it("should render", () => {
    const wrapper = shallow(<TodoListDataLayer />);
    expect(wrapper).toMatchSnapshot();
  });
});
