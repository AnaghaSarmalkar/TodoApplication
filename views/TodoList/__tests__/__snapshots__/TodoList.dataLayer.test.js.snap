// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TodoListDataLayer should render 1`] = `
<TodoList
  insets={{}}
  onItemPress={[Function]}
/>
`;
